includes/admin/class-sensei-setup-wizard-pages.php

```diff
@@ -103,35 +115,47 @@ private function get_learner_courses_page_content() {
 	 * @return string
 	 */
 	private function get_course_completed_page_content() {
-		$blocks   = [];
-		$blocks[] = serialize_block(
-			[
-				'blockName'    => 'core/paragraph',
-				'innerContent' => [ '<p class="has-text-align-center has-large-font-size">' . __( 'Congratulations on completing this course! 🥳', 'sensei-lms' ) . '</p>' ],
-				'attrs'        => [
-					'align'    => 'center',
-					'fontSize' => 'large',
-				],
-			]
-		);
-
-		$blocks[] = serialize_block(
-			[
-				'blockName'    => 'core/buttons',
-				'innerContent' => [ '<div class="wp-block-buttons is-content-justification-center"><!-- wp:button {"className":"more-courses"} --><div class="wp-block-button more-courses"><a class="wp-block-button__link">' . __( 'Find More Courses', 'sensei-lms' ) . '</a></div><!-- /wp:button --></div>' ],
-				'attrs'        => [ 'contentJustification' => 'center' ],
-			]
-		);
-
-		$blocks[] = serialize_block(
-			[
-				'blockName'    => 'sensei-lms/course-results',
-				'innerContent' => [],
-				'attrs'        => [],
-			]
+		$blocks = serialize_blocks(
+			/**
+			 * Filter the course completed page content when auto-creating it
+			 * through setup wizard.
+			 *
+			 * @hook  sensei_default_course_completed_page_template
+			 * @since 3.13.1
+			 *
+			 * @param {array} $blocks Blocks array.
+			 *
+			 * @return {array} Blocks array.
+			 */
+			apply_filters(
+				'sensei_default_course_completed_page_template',
```

Comment:
I think these filters could be shorter by removing the word `default`. I don't think it's really needed here.

Reply:
Hum. I think the _default_ word was important before [this change ](https://github.com/Automattic/sensei/pull/4286#discussion_r691524838), but as you said, probably with the new name we don't need the _default_. =)

Updated here: https://github.com/Automattic/sensei/pull/4290